package.json

```
@@ -12,7 +12,8 @@
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "lint": "tslint {src,test}/**/*.ts",
     "prebuild": "yarn clean",
-    "build": "tsc"
+    "build": "tsc -p tsconfig.json",
+    "prepublish": "tsc -p tsconfig.json"
```

Here could be just `"npm run build"` to keep consistent 👍 
```suggestion
    "prepublish": "npm run build"
```

---

Ok cool! I wasn't sure if `npm run build` will work inside another `npm` script! Great will do!